### DISCUSSION TITLE
Webrtc with aiohttp works but not fastapi

### DISCUSSION THREAD

**Discussion by pushkarprasad007** [fastapi/fastapi#11835] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
https://github.com/pushkarprasad007/llm-aiortc
```

### Description

I have a sample application (https://github.com/pushkarprasad007/llm-aiortc) which uses aiortc, where User is supposed to send the audio and then its recorded on python server on backend, and then python server is supposed to respond with 3 different audio back to back (mimicing LLM behavior where token by token are returned in streaming fashion). This project is intended to serve as a reference to lot of people who might be trying to achieve the conversational app using LLM.

There are two files to run

 1. server_aiohttp.py 
 2. server_fastapi.py

When I use python3.12, and I run the aiohttp code, it works fine. However, with fastapi, it gets stuck some place. Since a lot of development these days is in fastapi, it would be of great help if this project works with fastapi as well. I've spent enough time to debug, before deciding to come on this forum.

Any directions or better still, if someone can help figure, would be helpful not just for me but a whole lot of community who spent countless hours trying to make this work. I hope my earnest attempt to achieve this goes to a logical conclusion.

Steps:
```
python3.12 -m venv venv
source venv/bin/activate
pip install -r requirements.txt
python server_<aiohttp|fastapi>.py

Go to localhost:8080
Click on green power button
After some time Webrtc status shows 'connected'
Click on audio button and speak, and then click on stop.
```

### Operating System

Linux, macOS

### Operating System Details

Mac M1

### FastAPI Version

0.111.0

### Pydantic Version

Not using

### Python Version

3.10

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
